psl/psl/tests/validation/attributes/schema/on_mysql.prisma

```diff
@@ -22,9 +22,3 @@ model Test {
 // [1;94m 3 | [0m    url = env("TEST_DATABASE_URL")
 // [1;94m 4 | [0m    schemas = [1;91m["public", "sphere"][0m
 // [1;94m   | [0m
-// [1;91merror[0m: [1m@@schema is not supported on the current datasource provider[0m
```

Comment:
ℹ️  These tests changed as we now bail earlier in our validation logic as we deem the configuration invalid. The early return happens [here](https://github.com/prisma/prisma-engines/blob/eb9c75ddb11a0d1aba6036eb8073e2a43b015dd8/psl/psl-core/src/validate/validation_pipeline.rs#L60) due to the configuration validation [here](https://github.com/prisma/prisma-engines/blob/db5b07d7973eee997b44113b33f121339b48cfc5/psl/psl-core/src/lib.rs#L69).